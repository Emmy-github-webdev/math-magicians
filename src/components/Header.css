.nav-bar {
  background: #fff;
  display: flex;
  justify-content: space-around;
  align-items: center;
  font-size: 20px;
  height: 75px;
}

.nav-list {
  text-decoration: none;
  color: #000;
}
