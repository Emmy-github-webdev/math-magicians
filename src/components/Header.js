/* eslint-disable linebreak-style */
import React from 'react';
import { Link } from 'react-router-dom';
import './Header.css';

const Header = () => (
  <div className="nav-bar">
    <h1>Math Magicians</h1>
    <nav>
      <Link to="/" data-test="header-item" className="nav-list">Home</Link>
      {' '}
      |
      {' '}

      <Link to="/calculator" data-test="header-item" className="nav-list">Calculator</Link>
      {' '}
      |
      {' '}

      <Link to="/quote" data-test="header-item" className="nav-list">Quote</Link>

    </nav>
  </div>
);

export default Header;
