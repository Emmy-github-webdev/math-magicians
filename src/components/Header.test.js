import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from './Header';

it('renders correctly', () => {
  const Nav = renderer
    .create(<Router><Header /></Router>)
    .toJSON();
  expect(Nav).toMatchSnapshot();
});
