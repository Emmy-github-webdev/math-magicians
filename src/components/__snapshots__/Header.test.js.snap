// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="nav-bar"
>
  <h1
    data-test="header-item"
  >
    Math Magicians
  </h1>
  <nav>
    <a
      className="nav-list"
      data-test="header-item"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
     
    |
     
    <a
      className="nav-list"
      data-test="header-item"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
     
    |
     
    <a
      className="nav-list"
      data-test="header-item"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</div>
`;
